test/test-remote/looker/src/args.ts

```diff
@@ -141,6 +143,22 @@ export function parseCmdlineArgs(): void {
     default: 1
   });
 
+  parser.add_argument("--metrics-past-start-range-max-seconds", {
+    help:
+      "Starting amount in seconds to subtract from current wall time for metric series starts. " +
```

Comment:
We might want to try to make this easier to understand in a subsequent iteration.

A suggestion:

> Metrics time series start in the past compared to the initialization wall time. The exact time is picked randomly from an interval. This defines the lower bound of the interval.

Not perfect, but maybe a tiny little clearer?

Reply:
yeah that's a lot better. updated docs for both the min/max args